save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/demo/DemoRunRequest.kt

```diff
@@ -1,24 +1,27 @@
-package com.saveourtool.save.demo.diktat
+package com.saveourtool.save.demo
 
+import com.saveourtool.save.demo.diktat.DiktatDemoTool
 import kotlinx.serialization.Serializable
 
 /**
- * @property mode to tell if the analysis should be performed in [DiktatDemoMode.WARN] or in [DiktatDemoMode.FIX]
- * @property tool to tell if the analysis should be performed by [DiktatDemoTool.DIKTAT] or [DiktatDemoTool.KTLINT]
- * @property config additional configuration file for [DiktatDemoTool.DIKTAT]
+ * @property codeLines file as String that contains code requested for diktat demo run
+ * @property mode
+ * @property config
  */
 @Serializable
-data class DiktatAdditionalParams(
-    val mode: DiktatDemoMode = DiktatDemoMode.WARN,
-    val tool: DiktatDemoTool = DiktatDemoTool.DIKTAT,
-    val config: List<String> = defaultDiktatConfig,
+data class DemoRunRequest(
+    val codeLines: List<String>,
+    val mode: DemoMode?,
+    val config: List<String>?,
```

Comment:
Maybe make it `null` by default?

Reply:
Done

Reply:
no, you didn't get it
```suggestion
    val mode: DemoMode? = null,
    val config: List<String>? = null,
```

Reply:
Done